src/js/variant-widget/filters/eva-form-panel.js

```diff
@@ -193,5 +193,31 @@ EvaFormPanel.prototype = {
         });
         return form;
 
+    },
+    validatePositionFilter : function(values) {
+        var validate = true;
+        _.each(values, function (region) {
+            if(region.match(/^[\w\d]+\:\d+\-\d+/)){
+                var start = region.split(':')[1].split('-')[0];
+                var end = region.split(':')[1].split('-')[1]
+                if (end - start > 1000000) {
+                    Ext.Msg.alert('Limit Exceeds', 'Please enter the region no more than 1000000 range');
```

Comment:
Maybe something like "Please enter a region no larger than 1 million bases" could be a bit clearer?